pkg/tfgen/testdata/TestPlainDocsParser/table.golden

```diff
@@ -236,7 +236,7 @@ import com.pulumi.Context;
 import com.pulumi.Pulumi;
 import com.pulumi.core.Output;
 import com.pulumi.simple.resource;
-import com.pulumi.simple.ResourceArgs;
+import com.pulumi.simple.resourceArgs;
```

Comment:
It would appear that the javagen part of the pipeline is not receiving the metadata, which appears to be a pre-existing problem (see simple.resource which should have been simple.Resource), but now is also applicable to Args.